docs/docs/guides/integrations/bedrock.md

```diff
@@ -0,0 +1,118 @@
+# Amazon Bedrock
+
+Weave automatically tracks and logs LLM calls made via Amazon Bedrock, AWS's fully managed service that offers foundation models from leading AI companies through a unified API.
+
+## Traces
+
+Weave will automatically capture traces for Bedrock API calls. You can use the Bedrock client as usual after initializing Weave and patching the client:
+
+```python
+import weave
+import boto3
+from weave.integrations.bedrock.bedrock_sdk import patch_client
+
+weave.init("my_bedrock_app")
+
+# Create and patch the Bedrock client
+client = boto3.client("bedrock-runtime")
+patch_client(client)
```

Comment:
autopatching not possible?

Reply:
not really, as the client is dynamically create on the fly